frontend/src/features/properties/selector/components/MapClickMonitor.tsx

```diff
@@ -40,7 +40,7 @@ export const mapFeatureToProperty = (
     .getBounds()
     .getCenter();
   return {
-    pid: selectedFeature?.properties?.PID_NUMBER ?? undefined,
+    pid: selectedFeature?.properties?.PID ?? undefined,
```

Comment:
what is the rationale for this change?

Reply:
The definition of the IMapProperty defines the pid to be a string or undefined. Pid number was returning a number instead breaking the type promise for the interface and causing some issues downstream. i.e. checks for length of a string

Reply:
We should choose if we want to handle the pid as a string or a number in the frontend and abstract the conversions to the lowest level. That way we can have some guarantees when coding against that piece of info